src/app/routes/feed/stage.tsx

```diff
@@ -186,6 +193,32 @@ export default function FeedStage(): ReactNode {
 	// Normalize view.fields to maintain the canonical order defined in fields array
 	const normalizedView: ViewType = useMemo( () => normalizeFieldOrder( view, fields ), [ view, fields ] );
 
+	// Generate empty state content based on following status.
+	const renderEmptyStateContent = (): ReactNode => {
+		// If search or filters are active, show simple "no results" message.
+		if ( normalizedView.search || ( normalizedView.filters && normalizedView.filters.length > 0 ) ) {
+			return __( 'No posts found.', 'activitypub' );
+		}
+
+		// If not following anyone, show link to following page.
+		if ( followingResolved && ! followingCount ) {
+			const followingUrl =
+				activeActorId === 0
+					? addQueryArgs( 'options-general.php', { page: 'activitypub', tab: 'following' } )
+					: addQueryArgs( 'users.php', { page: 'activitypub-following-list' } );
+
+			return (
+				<>
+					{ __( 'Your feed is waiting to come alive.', 'activitypub' ) }{ ' ' }
+					<a href={ followingUrl }>{ __( 'Start following people on the Fediverse', 'activitypub' ) }</a>
```

Comment:
```suggestion
					{ createInterpolateElement(
						__(
							'Your feed is waiting to come alive. <a>Follow more people on the Fediverse</a> to see their posts here.',
							'activitypub'
						),
						{
							a: <a href={ followingUrl } />,
						}
					) }
```
Using `createInterpolateElement` would make this easier to translate for RTL languages.